### PR TITLE
Update bitnami postgres installation link

### PR DESCRIPTION
**Pull request by munkiat**
Currently copying the link from docs directly gives the following  helm install tyk-postgres bitnami/postgresql --set "auth.database=tyk_analytics" -n tyk --version 12.12.10 Error: INSTALLATION FAILED: repo bitnami not found

Edited to use the helm install link from https://github.com/bitnami/charts/tree/main/bitnami/postgresql

### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-charts/tyk-stack-chart.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ global:
 
 **PostgresSQL Installation**
 ```bash
-helm install tyk-postgres bitnami/postgresql --set "auth.database=tyk_analytics" -n tyk --version 12.12.10
+helm install tyk-postgres oci://registry-1.docker.io/bitnamicharts/postgresql --set "auth.database=tyk_analytics" -n tyk
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-charts/tyk-stack-chart.md:293]:
```suggestion
helm install tyk-postgres oci://registry-1.docker.io/bitnamicharts/postgresql --set "auth.database=tyk_analytics" -n tyk --version 16.2.3
```

I think we should restrict to the current version of chart, Have you tested with this [version](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/Chart.yaml#L38)?

**Review by sharadregoti — CHANGES_REQUESTED** [review]

File: tyk-docs/content/product-stack/tyk-charts/tyk-stack-chart.md (hunk 1 above)
Comment (munkiat) [reply tyk-docs/content/product-stack/tyk-charts/tyk-stack-chart.md:293]: @sharadregoti yes makes sense. I've used the latest; not sure if our automated tests uses v16.2.3 exactly. Would something like v15 work? https://tyk.io/docs/planning-for-production/database-settings/postgresql/

Comment (sharadregoti) [reply tyk-docs/content/product-stack/tyk-charts/tyk-stack-chart.md:293]: `16.2.3` is the helm chart version. In this chart version the default value for [postgres](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml#L112) is `17.2.0-debian-12-r2`. We need to select a chart version which uses postgres version `16.x` or `15.x` as per this [page](https://tyk.io/docs/planning-for-production/database-settings/postgresql/).

Comment (sharadregoti) [reply tyk-docs/content/product-stack/tyk-charts/tyk-stack-chart.md:293]: Or alternatively we can fixate on chat version `16.2.3` and set the image version `--set image.tag="16.6.0"`
I used this postgres [version](https://hub.docker.com/layers/bitnami/postgresql/16.6.0/images/sha256-af9927a44db2c3fcf145a0d6bf10541605f6f0b3fbf80f6db66d33c7fb4125ea?context=explore)

**Comment by sharadregoti** [comment]
I am closing this PR.

When you run the command and get the error `INSTALLATION FAILED: repo bitnami not found`. This signifies helm repository is not added locally. To resolve this just run this command `helm repo add bitnami https://charts.bitnami.com/bitnami` then run our command.

I have updated the command in this PR #5855.

Refer to the below image.
image

**Pull request closed without merging**